server/routes/signup.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const router = express.Router();
+const bcrypt = require("bcrypt");
+const saltRounds = 10;
+const User = require("../models/User");
+
+router.post("/:email/:password", (req, res, next) => {
+  const { email, password } = req.params;
+  let epCheck = false;
+  if (email.includes("@") && password.length > 6 && (epCheck = true)) {
```

Comment:
It's not good practice to put an assignment inside of a conditional block like this. You can use and else block to accomplish the same thing